setup.env

```diff
@@ -1,38 +1,63 @@
-### Change the LSST_SDK_INSTALL and OSPL_HOME to the actual locations and uncomment
-###
-### export LSST_SDK_INSTALL=/home/saluser
-### export OSPL_HOME=$LSST_SDK_INSTALL/OpenSpliceDDS/V6.9/HDE/x86_64.linux
+# Configuration file to setup SAL environment. Default config assumes
+# ts_opensplice is available in the same directory as ts_sal. You don't need to
+# edit this file for the default config described in README.md. Please consult
+# README.md for details.
```

Comment:
Consider removing the final sentence. It seems redundant.

Reply:
Removed. Thanks!